tests/test_other.py

```diff
@@ -497,19 +509,19 @@ def test_emcc_7(self):
         # The remainder of this test requires asmjs support
```

Comment:
Just noticed that the `em++` is not run if this is on wasm backend because the function returns...